torch/_decomp/decompositions.py

```diff
@@ -71,6 +71,14 @@ def _unsqueeze_to_dim(x: Tensor, dim: int):
     return x
 
 
+@register_decomposition(aten.conj_physical)
+@pw_cast_for_opmath
+def conj_physical(x: Tensor) -> Tensor:
+    if x.is_complex():
+        return x.real - x.imag
```

Comment:
probably should be `torch.complex(x.real, -x.imag)`

Reply:
Thanks, I thought x.real returns a complex tensor